### PR TITLE
Add Env Executor to dedicated

### PR DESCRIPTION
**Pull request by cbourgois**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/env-executor` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-4671
| License          | BSD 3-Clause

## Description

Add Env Executor to the `dedicated` application.

I had to refactor/update the build, to avoid multiple entrypoints in the `webpack.config.js`.

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/account/account.bundle.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import './user/user.routes';
+import './user/users/add/user-users-add.controller';
+import './user/users/delete/user-users-delete.controller';
+import './user/users/disable/user-users-disable.controller';
+import './user/users/enable/user-users-enable.controller';
+import './user/users/group.service';
+import './user/users/update/user-users-update.controller';
+import './user/users/users.controller';
+import './user/users/users.routing';
+import './user/users/users.service';
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/account/account.bundle.js]: Did you create a "moulinette" for this ? 😅

File: packages/manager/apps/dedicated/client/app/app.module.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+import './billing/autoRenew/actions/update/noPaymentMean/noPaymentMean.module';
+import './billing/autoRenew/actions/update/update.module';
+import './billing/autoRenew/actions/warnNicBilling/warnNicBilling.module';
+import './billing/autoRenew/actions/warnPendingDebt/pending-debt.module';
+import './billing/autoRenew/agreements/details/details.module';
+import './billing/autoRenew/agreements/user-agreements.module';
+import './billing/autoRenew/autorenew.module';
+import './billing/autoRenew/bulk/bulk.module';
+import './billing/autoRenew/disable-domains-bulk/disable-domains-bulk.module';
+import './billing/autoRenew/disable/disable.module';
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/app.module.js]: Why do we have to import all billing/autoRenew module ? Aren't they already imported by their parent ?

Comment (cbourgois) [reply packages/manager/apps/dedicated/client/app/app.module.js]: I just move `glob` from webpack config to javascript `import`. So it works like before.
I put them in `bundle` as it was previously named bundle in webpack. 
We have to clean theses files and avoid to import a list of js files.

File: packages/manager/apps/dedicated/client/app/account/account.bundle.js (hunk 1 above)
Comment (cbourgois) [reply packages/manager/apps/dedicated/client/app/account/account.bundle.js]: I wish I had...

**Comment by cbourgois** [comment]
I removed the previous `manager-preload` component and related translations for all languages.

File: packages/manager/apps/dedicated/client/app/index.html [hunk 3]
```diff
@@ -12,7 +12,6 @@
         <title>OVHcloud</title>
     </head>
     <body
-        data-ng-app="App"
         data-ng-controller="SessionCtrl as $ctrl"
         data-ng-cloak
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/index.html]: I have to move this one.

**Pull request merged**